client/src/competition/confirm-competition-delete-modal/ConfirmCompetitionDeleteModal.vue

```diff
@@ -0,0 +1,72 @@
+<template>
```

Comment:
There is `ConfirmDialog.vue` in the `components` folder. So, you can use that component instead of this. But I see I've done a stupid mistake and call it `ConfigDialog`

Reply:
Good point, I will do some of your suggestions and will create new pull request :)